### ISSUE TITLE
[SUGGESTION]: Opportunity not to participate in events.

### ISSUE THREAD

**Issue by xqmaareekpx** [pokeclicker/pokeclicker#3631]
### Current Behavior

I am new in this game and I guess everyone participates.

### Improved Behavior

Opportunity not to participate in events.

### Reason for change

Not everyone wants custom Pokemon. In a situation where everyone participates, we are somewhat forced into it.

### Detailed Description

Maybe a new challenge, or the ability to remove unwanted pokemon?

**Comment by Qwertypop04** [comment]
Just don't catch them then

**Comment by CooldudeOmega** [comment]
Don't catch them? This isn't something bad, you probably want more Pokémon, no reason not to get them, but if you don't want them just don't get them.

**Comment by xqmaareekpx** [comment]
It depends how You play. I might as well not play for a week. That's not what I'm talking about.

**Comment by DraKay** [comment]
Set new as not catch in ball selector easy peasy... This game have only 4 mandatory pokemons pidgey line and eternatus, anything else can be skipped. If you want to play only with vanilla pokemons there is no problem, just don't catch... If you can solve the problem without add more code why make a complex solution no needed

**Comment by xqmaareekpx** [comment]
I understand, but I would have to turn off all pokeballs, not just new (these pokemons have a shiny version?). This equates to not playing for a week.

**Comment by CooldudeOmega** [comment]
The more rational question is why do you dislike the eventmons so much, why do you not want to play for a week to avoid 3 Pokémon?

**Comment by EatPant2nd** [comment]
I mean, events usually don't happen in all regions so you can move to a different region.

**Comment by EatPant2nd** [comment]
No, I won't call it that. Also, yeah I can see that it would be a bit annoying to remove ball from "new" and "new shiny" for a week. Could be worse though.

**Comment by CooldudeOmega** [comment]
I'm confused, you want only official pokemon, in the pokemon fangame, I doubt that there'll be an opportunity to skip events but going through the trouble of programming this for easy to miss pokemon for a few people that don't like them seems, unproductive.

**Comment by DraKay** [comment]
Just avoid boosted route, not catching a single roamer is easy and if you are unlucky you save scum and done...

**Comment by xqmaareekpx** [comment]
[CooldudeOmega]
Maybe You're right. But firstly, it's a suggestion, and that's what they're for, and secondly, maybe more than a few, You do not know that.

Maybe the release option would be better?

[DraKay]
I know how to avoid catching pokemon by playing my way for a week. That's not my point at all.

:)

**Comment by DraKay** [comment]
Maybe for playing only a week you don't get that actually with your so new file is easier not to get the only "problematic" pokemon than get it, haven't make the maths but rest assure you probably would only need to reload a file once or rwice for this week

**Comment by xqmaareekpx** [comment]
I don't think You understand me.

Why challenge "no click attack", when You can just not click where You shouldn't? Each case can be treated this way.

**Comment by xslk** [comment]
The catch phrase for pokemon is gotta catch em all. A challenge to remove pokemon is counterintuitive to that logic. 

Old challenges of removing features are just still there for now. Any new challenges of removing new features have been denied and that's why it is a slow EV challenge instead of a no EV challenge. Devs deemed that removing features as a challenge would no longer be considered for any future challenges.

**Comment by CypherX** [comment]
No Click Attack is an actual, ya know, _challenge_. You having some weird vendetta against custom/event pokemon is not. The tools already exist to not catch things you don't want.

**Comment by xqmaareekpx** [comment]
"Catch them all." As I wrote, let's make 1000 more, it will be more fun. That's not what this suggestion is about. :)

It's not a vendetta, it's a suggestion.

Maybe it can be done with a simple question at the first entrance when the event starts "Do you want to participate or not?". 

"yes" - starts the event
"no" - doesn't change anything

Everything else I had to write on this subject, I have already written.

Everyone has the right to their opinion. I respect that and thank You all for the discussion.


I leave the decision to the developers.

**Comment by DraKay** [comment]
The real question is this: is worth to code something not as simple as you think for something that is not a real issue and only 1 person in years have ever asked?

**Comment by xslk** [comment]
Do you treat pokemon that appear in pokemon go with hats as custom pokemon? Do you also treat pokemon that appear in the anime of the movies as custom pokemon?

What is your criteria in determining if a pokemon is custom or not?

**Comment by DapperDwarf** [comment]
I have seen people complain about toggles similar to what you're suggesting in Halo: MCC, that game collection added armour sets not in the games it's based on and has a toggle to not see them but people complained that their favourite new armours should have an option to be added without their least favourite ones there.  you could just use the "category" feature already in the game to not see pokemon you don't want to see (in the breeding menu at least) if they bother you that much

**Comment by xqmaareekpx** [comment]
I get the impression that You want to convince me that this suggestion is pointless. I'm sorry, it makes sense to me. :)


[DraKay]
Just because someone hasn't written something doesn't mean they don't want it. I know a few other people who feel the same way as me. I don't know if it's hard to do or not, so I leave the decision to the developers.

[xslk]
Basically everything is here: https://www.pokemon.com/us/pokedex
For me, just because You dress up a Pokemon doesn't mean it's a different Pokemon.

[DapperDwarf]
They don't bother me. I just wish I had the option to not catch them without making my game 'harder' (thinking when and where the event is, what to disable etc) or the option to remove them once they get caught.

**Comment by DraKay** [comment]
I love the silent mayory argument... Maybe people don't write it, but I really know a lot of people who think I'm a goddess who could destroy the world with a blink... Sorry.... But maybe I have a too scientific mentality, but if I have no proofs of something I don't belive it at all...

 And reading your answer to slk maybe this game is gonna be too stressful for you... This is an event with only 3 pokemons, but we have a full su region with 35 pokemons take from the anime...

**Comment by Qwagsoir** [comment]
It's drakay he always think he's better than literally everyone else lmao, probably the biggest turnoff for anyone interacting with the community

**Comment by Aegyo** [comment]
Approved, a setting to disable events should be fine.
